typescript-client/introduction.mdx

```diff
@@ -17,3 +17,12 @@ import { LiteralClient } from "@literalai/client";
 const client = new LiteralClient(process.env["LITERAL_API_KEY"]);
 ```
 
+#### Disable Decorators
```

Comment:
we do not have decorators in the typescript client. It disables the thread and step ingestion apis like thread.step()